libnmstate/metadata.py

```diff
@@ -198,6 +194,13 @@ def _generate_link_master_metadata(
                         )
 
 
+def _iface_to_remove(iface_state):
+    return iface_state.get(Interface.STATE) not in (
```

Comment:
Do you know why this uses get? The state is mandatory, isn't it? It needs to be set explicitly or it will always be merged in from the current state, wouldn't it?

Reply:
The state is not a must, it will use the current.